src/libraries/Microsoft.PowerFx.Core/Public/Values/BuiltinFormulaTypeConversions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Public.Types;
+
+namespace Microsoft.PowerFx.Core
+{
+    /// <summary>
+    /// Mapping between builtin dotnet Types and power fx types.
+    /// </summary>
+    public static class BuiltinFormulaTypeConversions
```

Comment:
Do we expect this to be useful publicly?

Reply:
Yes.  I wanted it for the SqlRunner, help in marshalling Sql results to Formula Value.